docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
 
-At each block, the top 100 validators who are not jailed are said to be bonded.
+Various possibilities are considered and ruled out. In short, Lamport, Shostak, and Pease found that success was only assured when honest generals outnumbered traitors (**m**) by a number greater than **3m + 1** traitors. In other words, coordination or consensus is impossible to achieve if a third or more of the generals are traitors.
 
-If, for any reason, a validator goes outside the bonded validators set, it enters an unbonding period.
-A validator that has failed to get itself back into the bonded validators set within 21 days is said to be unbonded.
-The number 100 can be updated via a simple governance proposal. – Need testing and verification.
+:::
 
-**Only bonded validators can propose or sign blocks.
-Only bonded validators can earn block rewards.
-Only bonded validators’ tokens are subject to slashing.**
+### Incentivisation
 
-Although the actual act of slashing might occur during the unbonding period.
+Second, incentivisation is built in to the tokenomics model to encourage good behaviour. **Validator operators** who secure the network by running a **Validator node** are required to **stake or self-delegate** a minimum of **2M CUDOS** in return for network rewards. Staking or self-delegating higher amounts increases the chances of that Validator being selected to perform block validation as well as increasing their rewards (as a proportion of the overall amount staked on the network). 
 
-## Redelegation 
+Validators and Delegators earn block rewards for keeping the blockchain operational and secure.
+**Block rewards** consist of **Staking** rewards and **Transaction fees**.
 
-A **delegator** can choose to undelegate their tokens at any time.
 
-A **delegator** can redelegate their tokens from **Validator A** to **Validator B** without any penalties or extra waiting times. 
+### Validator self-staking
 
-To redelegate tokens from **Validator B** to **Validator C** you must wait 21 days. 
+:::info Terminology
 
-However, they need to wait for 21 days if they want to further redelegate any amount of tokens from Validator B – irrespective of how much they redelegated to Validator B initially.
+**Staking** tokens is also referred to as **delegating**.
 
-A validator can **self-delegate only through the CLI** – explorer v1 doesn’t provide this functionality.
+:::
 
-A validator can also have an **additional stake added** to it via the CLI to increase its voter power.
+A **Validator Operator** must **self-delegate** at least 2M CUDOS to a node for it to be operational. They can increase their chances of attracting delegations from token holders by running a highly reliable node with excellent uptime. This in turn increases their chances of being selected to issue a block. 
```

Comment:
I'd remove the "to a node" part because it sounds like ot could be any node, but it's not because is self-delegating.

Reply:
changed to "their node"